src/hooks/useAnnouncementsUploads.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react'
+import { useClient } from 'cozy-client'
+import { AnnouncementsConfig } from 'components/Announcements/types'
+import flag from 'cozy-flags'
+
+const useAnnouncementsUploads = (url: string | undefined): string | null => {
```

Comment:
nit: My understanding is that `upload` is a name relative to announcements but does not means that we are uploading anything here? Would be nice to have some nomenclature to explain this somewhere

Reply:
We are not uploading anything we only get images from the announcements server which are located inside the uploads route. To avoid misunderstanding I added some documentation and also renamed the hook to `useAnnoncementsImage`